test/e2e/csv_e2e_test.go

```diff
@@ -4249,11 +4258,19 @@ var _ = Describe("Disabling copied CSVs", func() {
 
 			return nil
 		}).Should(Succeed())
+		Eventually(func() error {
+			err := ctx.Ctx().Client().Delete(context.Background(), &og)
+			if err != nil && k8serrors.IsNotFound(err) {
+				return err
+			}
+
+			return nil
+		}).Should(Succeed())
```

Comment:
I was unaware that failing to delete the OperatorGroup within a namespace prior to deleting the namespace could cause timeout issues. This is unintuitive to me given that OperatorGroups are a namespaced resource that shouldn't require much overhead to delete, do we know the cause?

Reply:
Actually I don't know if the OperatorGroup is causing the error.  I delete it to eliminate all possible cause of the namespace deletion hung. It is the only resource remaining in the namespace except default service accounts and secrets that all namespace have.

Reply:
Hopefully when we integrate the new client, resource clean-up will be ezpz!